subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -23,6 +23,22 @@
             "changes": [
                 "org.gradle.kotlin.dsl.support.delegates.SettingsDelegate.include(java.lang.String[])"
             ]
+        },
+        {
+            "type": "org.gradle.api.initialization.Settings",
+            "member": "Method org.gradle.api.initialization.Settings.includeFlat(java.lang.Iterable)",
+            "acceptation": "Adding new Settings#includeFlat(Iterable)",
+            "changes": [
+                "Method added to interface"
+            ]
+        },
+        {
+            "type": "org.gradle.api.initialization.Settings",
+            "member": "Method org.gradle.api.initialization.Settings.includeFlat(java.lang.String[])",
+            "acceptation": "Adding new Settings#includeFlat(Iterable)",
+            "changes": [
+                "METHOD_ABSTRACT_NOW_DEFAULT"
```

Comment:
Is this a good message here? I thought we used plain English.

Reply:
I got it from https://github.com/gradle/gradle/commit/e74330dac12b29b923872bc42670cfb64d50eaa4